db/schema.rb

```diff
@@ -247,10 +247,10 @@
     t.string "scopes", default: "", null: false
     t.string "lead_application_source", default: "", null: false
     t.boolean "confidential", default: true, null: false
-    t.boolean "can_access_private_user_data", default: false, null: false
-    t.boolean "can_find_or_create_accounts", default: false, null: false
-    t.boolean "can_message_users", default: false, null: false
-    t.boolean "can_skip_oauth_screen", default: false, null: false
+    t.boolean "can_access_private_user_data", default: false
+    t.boolean "can_find_or_create_accounts", default: false
+    t.boolean "can_message_users", default: false
+    t.boolean "can_skip_oauth_screen", default: false
```

Comment:
I don't know why the `null: false` went away. I think this is not correct. You might've ran a migration at one point which modified your local database, and then switched back to your branch, ran another migration and the changes from the other branch remained? Maybe? I think you can fix this by simply reverting these lines in your schema.

Reply:
Fixed